tests/tests_integration/test_api/test_transformations/test_transformations.py

```diff
@@ -155,3 +173,22 @@ def test_preview(self, cognite_client):
     def test_preview_to_string(self, cognite_client):
         query_result = cognite_client.transformations.preview(query="select 1 as id, 'asd' as name", limit=100)
         dumped = str(query_result)
+
+    def test_update_dmi_alpha(self, cognite_client, new_transformation):
+        default_api_subversion = cognite_client.config.api_subversion
+        new_transformation.destination = AlphaDataModelInstances("myTest")
+        with pytest.raises(NotImplementedError):
+            cognite_client.transformations.update(new_transformation)
+        partial_update = TransformationUpdate(id=new_transformation.id).destination.set(
+            AlphaDataModelInstances("myTest2")
+        )
+        with pytest.raises(NotImplementedError):
+            cognite_client.transformations.update(partial_update)
+
+        cognite_client.config.api_subversion = "alpha"
+        updated_transformation = cognite_client.transformations.update(new_transformation)
+        assert updated_transformation.destination == AlphaDataModelInstances("myTest")
+        # TODO: Fix partial update
+        # partial_updated = cognite_client.transformations.update(partial_update)
+        # assert partial_updated.destination == AlphaDataModelInstances("myTest2")
```

Comment:
cannot serialize `AlphaDataModelInstances` when performing partial updates, investigating it...